README.md

```diff
@@ -1,25 +1,39 @@
 Riak .NET Client - a .NET client for Riak
 =========================================
 
-**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://riak.basho.com/), [Basho](http://www.basho.com/)'s fault-tolerante, distributed KV store.
-
+**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://basho.com/riak/), an open source, distributed database that focuses on high availability, horizontal scalability, and *predictable*
+latency. Both Riak and this code is maintained by [Basho](http://www.basho.com/).
+
+1. [Installation](#installation)
```

Comment:
As with "Contributing", this section should link to the wiki:

https://github.com/basho/riak-dotnet-client/wiki/Installation

and the contents here should be merged with / overwrite what is in the wiki.